src/components/AccountCircleButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import AccountCircle from 'src/icons/AccountCircle'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+  size?: number
+  testID?: string
+}
+
+export default function AccountCircleButton({ style, size, testID }: Props) {
+  const onPress = () => {
+    navigate(Screens.ProfileMenu)
```

Comment:
should we add an analytics event for this?

Reply:
Added in c4cf8fe45442b62faaf8bbba250a11a5a5f8d0ad!